File: QuarkusLokiLogger/src/main/java/net/codinux/log/loki/quarkus/QuarkusLokiLogAppender.java
package net.codinux.log.loki.quarkus;

import net.codinux.log.loki.JBossLoggingLokiAppender;
import net.codinux.log.loki.quarkus.config.QuarkusLokiLogAppenderConfig;
import net.codinux.log.loki.quarkus.config.mapper.QuarkusLokiConfigMapper;

public class QuarkusLokiLogAppender extends JBossLoggingLokiAppender {

    public QuarkusLokiLogAppender(QuarkusLokiLogAppenderConfig config) {
        super(QuarkusLokiConfigMapper.mapConfig(config));
    }

}

File: QuarkusLokiLogger/src/main/kotlin/net/codinux/log/loki/quarkus/QuarkusLokiLogAppender.kt
package net.codinux.log.loki.quarkus

import net.codinux.log.LogAppenderConfig
import net.codinux.log.loki.LokiJBossLoggingAppender
import net.codinux.log.loki.quarkus.config.QuarkusLokiLogAppenderConfig

class QuarkusLokiLogAppender(quarkusLokiConfig: QuarkusLokiLogAppenderConfig) : LokiJBossLoggingAppender(mapConfig(quarkusLokiConfig)) {

    companion object {

        private fun mapConfig(config: QuarkusLokiLogAppenderConfig): LogAppenderConfig {

            return LogAppenderConfig()
        }

    }

}